USER_MANUAL.md

```diff
@@ -509,7 +469,7 @@ This section describes features on Tools-Filter.
 Control | Description
  -------------------------- | ------------------------------------------------------------------------ |
 Noise Suppression | Enable noise suppression, de-reverberation, AGC of mic signal using the Speex pre-processor
-700C/700D Auto EQ | Automatic equalization for FreeDV 700C and FreeDV 700D Codec input audio
+700D Auto EQ | Automatic equalization for FreeDV 700D/E Codec input audio
```

Comment:
I thought auto EQ was only for 700D but I guess it makes sense that it'd work for E too.